__tests__/__snapshots__/App-test.js.snap

```diff
@@ -5170,7 +5170,7 @@ exports[`Expander Example Page 1`] = `
                       undefined,
                     ]
                   }
-                  timeZoneOffsetInSeconds={-0}
+                  timeZoneOffsetInSeconds={-28800}
```

Comment:
What about this change changed the time zone? 🤣

Reply:
This has been happening on a few of my previous gallery PRs, talked to @chiaramooney about it and there seemed to be a difference in the timezone between the CI checks and the local tests - just need to manually change the value in the snapshot file locally.

Reply:
@chiaramooney is it possible to omit certain properties from the snapshots?

Reply:
To my knowledge, no there isn't. @YajurG if you want to give a quick skim of the Jest Snapshot test docs to confirm that could be good. But yes there does seem to be a mismatch between our local time settings and the CI machines.